### PR TITLE
App description updates and moving the about the app section from the bottom of the page to the top of the page

### PR DESCRIPTION
**Pull request by mirabalani**
App description updates and moving the about the app section from the bottom of the page to the top of the page

### CODE REVIEW

File: examples/asset-search/README.md [hunk 1]
```diff
@@ -8,6 +8,13 @@
   If you prefer, you can install and use equivalent alternatives, such as `yarn` or `pnpm`.
 - For more information about implementing [drag and drop](https://developers.miro.com/docs/add-drag-and-drop-to-your-app), visit our [developer documentation](https://developers.miro.com).
 
+### About the app
+
+This sample app shows how to implement asset searching and filtering by name and tags. \
+This app uses React and [Vite](https://vitejs.dev/), and it's based on the [`create-miro-app`](https://www.npmjs.com/package/create-miro-app) React/Typescript starter.
```
Comment (mettin) [inline examples/asset-search/README.md:14]:
```suggestion
This app uses React and [Vite](https://vitejs.dev/), and it's based on the [`create-miro-app`](https://www.npmjs.com/package/create-miro-app) React/TypeScript starter.
```

File: examples/drag-and-drop/README.md [hunk 2]
```diff
@@ -8,6 +8,13 @@
   If you prefer, you can install and use equivalent alternatives, such as `yarn` or `pnpm`.
 - For more information about implementing [drag and drop](https://developers.miro.com/docs/add-drag-and-drop-to-your-app), visit our [developer documentation](https://developers.miro.com).
 
+### About the app
+
+This sample app shows how to implement image drag and drop to a Miro board. \
+This app uses React and [Vite](https://vitejs.dev/), and it's based on the [`create-miro-app`](https://www.npmjs.com/package/create-miro-app) React/Typescript starter.
```
Comment (mettin) [inline examples/drag-and-drop/README.md:14]:
```suggestion
This app uses React and [Vite](https://vitejs.dev/), and it's based on the [`create-miro-app`](https://www.npmjs.com/package/create-miro-app) React/TypeScript starter.
```

File: examples/github-appcards/README.md [hunk 3]
```diff
@@ -12,6 +12,14 @@ This full-stack example shows how to build an integration with GitHub that syncs
   If you prefer, you can install and use equivalent alternatives, such as `yarn` or `pnpm`.
 - For more information about implementing [drag and drop](https://developers.miro.com/docs/add-drag-and-drop-to-your-app), visit our [developer documentation](https://developers.miro.com).
 
+### About the app
+
+This full-stack sample app shows how to build an integration with GitHub Projects that enables a 2-way sync data flow between GitHub issues and Miro app cards.
+
+This app uses React and [Vite](https://vitejs.dev/), and it's based on the [`create-miro-app`](https://www.npmjs.com/package/create-miro-app) React/Typescript starter.
```
Comment (mettin) [inline examples/github-appcards/README.md:19]:
```suggestion
This app uses React and [Vite](https://vitejs.dev/), and it's based on the [`create-miro-app`](https://www.npmjs.com/package/create-miro-app) React/TypeScript starter.
```

File: examples/nextjs-full-stack/README.md [hunk 4]
```diff
@@ -1,8 +1,10 @@
-# Full-stack Miro app example
 
-This sample app demonstrates the use of the Miro REST API and Web SDK. After configuring your project outlined in the Prerequisites section, this app allows you to authenticate yourself with your Miro account, and use the related access tokens to make REST API calls via the Miro Developer Platform.
+## About the app
+
+This full-stack sample app shows how to build a Next.js application that uploads a camera image to a Miro board by using the Miro Web SDK and Miro REST API. This app demonstrates the use of the Miro REST API and Web SDK. After configuring your project outlined in the Prerequisites section, this app allows you to authenticate yourself with your Miro account, and use the related access tokens to make REST API calls via the Miro Developer Platform.
```
Comment (mettin) [inline examples/nextjs-full-stack/README.md:4]: Why do we use `sample` in favour of `example`?

File: examples/nextjs-oauth/README.md [hunk 5]
```diff
@@ -1,8 +1,10 @@
-# Full-stack Miro app example
+# About the app
+
+This full-stack sample app shows how to implement the OAuth 2.0 authorization code grant flow from Miro to a client-side application built with Next.js.
 
 This sample app demonstrates the use of the Miro REST API and OAuth authorization. After configuring your project outlined in the Prerequisites section, this app allows you to authenticate yourself with your Miro account, and use the related access tokens to make REST API calls via the Miro Developer Platform.
 
-This example also stores your access and refresh tokens in your browsers cookies to demonstrate the Miro's REST API. When storing tokens in your own app, you should be consider encrypting or storing your user's tokens in a more secure way.
+This app also stores your access and refresh tokens in your browsers cookies to demonstrate the Miro's REST API. When storing tokens in your own app, you should be consider encrypting or storing your user's tokens in a more secure way.
```
Comment (mettin) [inline examples/nextjs-oauth/README.md:7]: On the previous line we say `this sample app`, now it says `this app`. Should we harmonize?

File: examples/wordle/README.md [hunk 6]
```diff
@@ -8,6 +8,16 @@
   If you prefer, you can install and use equivalent alternatives, such as `yarn` or `pnpm`.
 - For more information, visit our [developer documentation](https://developers.miro.com).
 
+### About the app
+
+This sample app shows you how to build the famous Wordle game in Miro. \
+The game opens the Library panel allowing you to start a new game and enter your guesses. \
+The game creates sticky notes and colors them the same way Wordle does it.
+
+This app uses React and [Vite](https://vitejs.dev/), and it's based on the [`create-miro-app`](https://www.npmjs.com/package/create-miro-app) React/Typescript starter.
```
Comment (mettin) [inline examples/wordle/README.md:17]:
```suggestion
This app uses React and [Vite](https://vitejs.dev/), and it's based on the [`create-miro-app`](https://www.npmjs.com/package/create-miro-app) React/TypeScript starter.
```

File: examples/wordle/package.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "miro-wordle",
-  "description": "This sample app shows how to build the famous Wordle game in Miro, where sticky notes are created and color-coded based on Wordle game rules. This app uses React and Vite, and it's based on the 'create-miro-app' React/Typescript starter.",
+  "description": "Build the famous Wordle game in Miro, where sticky notes are created and color-coded based on Wordle game rules. This app uses React and Vite, and it's based on the 'create-miro-app' React/Typescript starter.",
```
Comment (mettin) [inline examples/wordle/package.json:3]:
```suggestion
  "description": "Build the famous Wordle game in Miro, where sticky notes are created and color-coded based on Wordle game rules. This app uses React and Vite, and it's based on the 'create-miro-app' React/TypeScript starter.",
```

**Review by mettin — CHANGES_REQUESTED** [review]

File: examples/nextjs-full-stack/README.md (hunk 4 above)
Comment (mirabalani) [reply examples/nextjs-full-stack/README.md:4]: good point - should be example not sample here - we should get rid of example or sample in all descriptions though coz itz redundant will check and update all instances - thanks!

**Review by mettin — APPROVED** [review]

**Pull request merged**